test/src/main/java/org/apache/accumulo/test/fate/FateInterleavingIT.java

```diff
@@ -386,10 +407,10 @@ private FateId verifySameIds(Iterator<Entry<Key,Value>> iter, SortedMap<Key,Valu
     Text fateId = subset.keySet().iterator().next().getColumnFamily();
     assertTrue(subset.keySet().stream().allMatch(k -> k.getColumnFamily().equals(fateId)));
 
-    var expectedVals = Set.of("FirstNonInterleavingOp::isReady", "FirstNonInterleavingOp::call",
+    var expectedVals = List.of("FirstNonInterleavingOp::isReady", "FirstNonInterleavingOp::call",
```

Comment:
The use of a List here is really subtle in its importance to verify the correctness of the observed data.  A comment would be good to make its importance more prominent. 

```suggestion
    // In addition to ensuring the expected fate operations happened also want to ensure they happened in a specific order, this why a List was used.
    var expectedVals = List.of("FirstNonInterleavingOp::isReady", "FirstNonInterleavingOp::call",
```